lib/spack/spack/fetch_strategy.py

```diff
@@ -290,6 +281,16 @@ def mirror_id(self):
         return os.path.sep.join(
             ['archive', self.digest[:2], self.digest])
 
+    @property
```

Comment:
Should we move this function back to the same location? It  might be easier to see what has changed